homeassistant/components/prowl/const.py

```diff
@@ -4,3 +4,4 @@
 
 DOMAIN = "prowl"
 PLATFORMS = [Platform.NOTIFY]
+YAML_DEPRECATED_IN = "2026.1.0"
```

Comment:
Not sure how to choose when it should be deprecated. Very happy for suggestions for this one.